modules/release-notes/pages/sdk-release-notes.adoc

```diff
@@ -95,7 +119,7 @@ This new support for multiple identities enables you to test role-based access c
 ** Language support to enable canisters to take installation arguments.
```

Comment:
A parameterized actor *class* defines a canister that takes an argument on installation.